wayfinder/setup.py

```diff
@@ -0,0 +1,76 @@
+from codecs import open  # To use a consistent encoding
+from os import path
+
+from setuptools import setup
+
+HERE = path.dirname(path.abspath(__file__))
+
+# Get version info
+ABOUT = {}
+with open(path.join(HERE, 'datadog_checks', 'wayfinder', '__about__.py')) as f:
+    exec(f.read(), ABOUT)
+
+# Get the long description from the README file
+with open(path.join(HERE, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
+def get_dependencies():
+    dep_file = path.join(HERE, 'requirements.in')
+    if not path.isfile(dep_file):
+        return []
+
+    with open(dep_file, encoding='utf-8') as f:
+        return f.readlines()
+
+
+def parse_pyproject_array(name):
+    import os
+    import re
+    from ast import literal_eval
+
+    pattern = r'^{} = (\[.*?\])$'.format(name)
+
+    with open(os.path.join(HERE, 'pyproject.toml'), 'r', encoding='utf-8') as f:
+        # Windows \r\n prevents match
+        contents = '\n'.join(line.rstrip() for line in f.readlines())
+
+    array = re.search(pattern, contents, flags=re.MULTILINE | re.DOTALL).group(1)
+    return literal_eval(array)
+
+
+CHECKS_BASE_REQ = parse_pyproject_array('dependencies')[0]
+
+
+setup(
+    name='datadog-wayfinder',
+    version=ABOUT['__version__'],
+    description='The wayfinder check',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    keywords='datadog agent wayfinder check',
+    # The project's main homepage.
+    url='https://github.com/DataDog/integrations-extras',
+    # Author details
+    author='Jonny Scott',
+    author_email='jonny.scott@appvia.io',
```

Comment:
These should probably be updated with whatever values wind up in the manifest `author` field based on Ana's notes.

Reply:
done